static/app/types/hooks.tsx

```diff
@@ -646,6 +647,20 @@ type InviteButtonCustomizationHook = () => React.ComponentType<{
   organization: Organization;
 }>;
 
+/**
+ * Sidebar navigation item customization allows passing render props to disable
+ * the link, wrap it in an upsell modal, and give it some additional content
+ * (e.g., a Business Icon) to render.
+ */
+type SidebarNavigationItemHook = () => React.ComponentType<{
+  children: (opts: {
+    AdditionalContent: React.ReactElement | null;
```

Comment:
Why is this capitalized

Reply:
My bad, it was originally also a renderable, like `Wrapper`! I changed it, but didn't lowercase. I'll update 👍🏻